advisories/a90146.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Technical Advisory 90146
+advisory: A-90146
+summary: Changefeeds using the <code>initial_scan='only'</code> or <code>schema_change_policy='stop'</code> options may incorrectly complete with a <code>successful</code> status under certain circumstances.
+toc: true
+affected_versions: v22.1.6 to v22.1.9
+advisory_date: 2022-12-19
+docs_area: releases
+---
+
+Publication date: {{ page.advisory_date | date: "%B %e, %Y" }}
+
+## Description
+
+In CockroachDB [v22.1.6](../releases/v22.1.html#v22-1-6) to [v22.1.9](../releases/v22.1.html#v22-1-9), changefeeds using the [`initial_scan='only'`](../v22.1/create-changefeed#initial-scan) option can complete with a `successful` status even if not all messages have been emitted to the sink. This occurs consistently unless the table is very small.
+
+The same issue may occur if a changefeed with [`schema_change_policy='stop'`](../v22.1/create-changefeed#options) encounters a schema change. The changefeed may stop prior to all changes that occurred prior to the schema change being emitted.
```

Comment:
"prior to all changes that occurred prior to" is a little hard for me to parse, due to the repetition of "prior". Would a word substitution and/or rephrase here help?